src/components/Selection.js

```diff
@@ -14,6 +14,7 @@ export default function Selection({ selectedProject, setSelectedProject }) {
         <svg
           xmlns="http://www.w3.org/2000/svg"
```

Comment:
I would **highly** recommend to outsource your options array and import it here. It would make this component much more readable and easier to understand!

Also, we try not to pass the setter function of a state (setSelectedProject) as props directly but pass a callback function. It might lead to issues when passing the setter function of a state and it's not advised by the react guys. (Unfortunately I cannot comment there, that's why you have this random comment here 😅 )